schema/2.0/model/cyclonedx-blueprint-2.0.schema.json

```diff
@@ -340,6 +357,8 @@
             "compliance": "Regulatory compliance perspective",
             "risk": "Risk management perspective",
             "stakeholder": "General stakeholder perspective",
+            "use-case": "A high level persepctive that captures the data-flows of the use case, rather than deep granularity of systems",
+            "abuse-case": "A high level persepctive that captures the data-flows of a certain abuse case, rather than deep granularity of systems",
```

Comment:
Remove use-case as this is covered in the usecase schema. Add abuse case to the threat schema